ns-provisioner-samples/gitops-airgap/resources/settings-xml.yaml

```diff
@@ -0,0 +1,30 @@
+#@ load("@ytt:data", "data")
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: settings-xml
+type: service.binding/maven
+stringData:
+  type: maven
+  provider: sample
+  #@yaml/text-templated-strings
+  settings.xml: |
+    <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+        <mirrors>
+            <mirror>
+                <id>reposilite</id>
+                <name>Tanzu seal Internal Repo</name>
```

Comment:
Should we make this name something more generic?

Reply:
Changed to `Accelerator samples`